modules/preview-react/side-panel/stories/SidePanel.stories.mdx

```diff
@@ -112,7 +110,7 @@ is an example:
 
 #### Props
 
```

Comment:
This doubles up, so we can remove it.
<img width="740" alt="Screenshot 2023-02-23 at 1 21 12 AM" src="https://user-images.githubusercontent.com/1075861/220854926-5ad49c92-94ec-49f5-897f-d1e2a5f3a192.png">


```suggestion
```